sos/collector/clusters/ocp.py

```diff
@@ -122,6 +124,41 @@ def check_enabled(self):
         _who = self.fmt_oc_cmd('whoami')
         return self.exec_primary_cmd(_who)['status'] == 0
 
+    def create_clusterrolebinding(self, namespace):
```

Comment:
What's the need for this? Is there some deployment/config that would cause the existing approach to fail?